cmd/manager/main.go

```diff
@@ -123,10 +147,6 @@ func main() {
 		os.Exit(1)
 	}
 
-	if systemNamespace == "" {
-		systemNamespace = util.PodNamespace()
-	}
-
```

Comment:
Why this?

Reply:
Looks like it was moved higher: https://github.com/operator-framework/operator-controller/pull/825/files#diff-435eecb1d2af40ee96747f88ab71eb2758da989c92f76dcb1f714fc1b300c633R89-R91